metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -147,6 +149,11 @@ def get_details_from_env() -> Tuple[Optional[str], Optional[str]]:
         ENV_METADATA_PORT_DEPRECATED
     )
     token = os.environ.get(ENV_METADATA_TOKEN)
+    if token is None:
+        system_client_id = os.environ.get(ENV_DATAHUB_SYSTEM_CLIENT_ID)
+        system_client_secret = os.environ.get(ENV_DATAHUB_SYSTEM_CLIENT_SECRET)
```

Comment:
Where are these being set for the actions pod? That's what we'll need right?

Reply:
These were all being set already. They are being used in actions pod recipes already. They were just undocumented variables not used in our CLI